requirements.txt

```diff
@@ -0,0 +1,11 @@
+fastapi
+uvicorn 
+pydantic
+GitPython 
+python-dotenv
+tinydb 
+nox 
+pytest 
+requests 
+black
+isort 
```

Comment:
Include versions used.

Reply:
Can you use only poetry rather than requirements.txt for the pip tool?

Reply:
> Can you use only poetry rather than requirements.txt for the pip tool?

Sure thing, Thanks.